zubhub_backend/zubhub/activities/tests/test_utils.py

```diff
@@ -0,0 +1,223 @@
+from django.test import TestCase
+from activities.models import * #import all the models
+from activities.utils import * #import all the utilities
+
+class TestUtils(TestCase):
```

Comment:
I truly appreciate your feedback on this. I understand the idea behind what you've said. Earlier on, I also considered doing this, but a slight problem with this approach is that every class will need to have its own setup (to avoid dependence on another class). This would lead to a significant amount of repetition, and it would also be harder to maintain in the long run: for example, if something is changed about how activities or images are created in the `utils.py`, every test class that has an activity or image in its setup would have to be modified accordingly.

Still following your approach though, would it be better if I grouped into two classes: `class TestActivityOperations` and `TestImageOperations`? It would allow shared setups.
This would also prevent making classes out of minor `util` functions